site/en/install/docker.md

```diff
@@ -130,7 +130,7 @@ lspci | grep -i nvidia
 Verify your `nvidia-docker` installation:
 
 <pre class="devsite-terminal devsite-click-to-copy">
-docker run --gpus all --rm nvidia/cuda nvidia-smi
+sudo docker run --rm --runtime=nvidia --gpus all ubuntu nvidia-smi
```

Comment:
I would not include `sudo` to prevent users running with unneeded high privileges.

```suggestion
docker run --rm --runtime=nvidia --gpus all ubuntu nvidia-smi
```